### PR TITLE
Update documentation about MAF

### PR DESCRIPTION
**Pull request by sandertan**
# What? Why?
- Clarified how to make a minimal MAF file
- Clarified how to make an extended MAF file with vcf2maf
- Clarified how to use custom driver annotations
- Restructured and numbered the MAF columns for minimal MAF file.
- Improved structure of Mutation Data section
- Updated screenshot for custom driver annotations (OncoPrint in v1.11.2)

### CODE REVIEW

File: docs/File-Formats.md [hunk 1]
```diff
@@ -549,39 +547,38 @@ profile_name: Mutations
...
-* **HGVSp_Short**: (annotation column) Amino Acid Change, e.g. p.V600E.
+### Data file
+The mutation data file extends the [Mutation Annotation Format](https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification) (MAF) created as part of the [Cancer Genome Atlas](https://wiki.nci.nih.gov/display/TCGA/TCGA+Home) project, by adding *extra annotations* to each mutation record. This section describes two types of MAF files:
```
Comment (dionnezaal) [inline docs/File-Formats.md]: Link to The Cancer Genome Atlas should include the 'The'

File: docs/File-Formats.md [hunk 2]
```diff
@@ -549,39 +547,38 @@ profile_name: Mutations
...
+1. A minimal MAF file with only the columns required for cBioPortal.
+2. An extended MAF file created with [vcf2maf or maf2maf](https://github.com/mskcc/vcf2maf).
```
Comment (dionnezaal) [inline docs/File-Formats.md]: Add line below that columns from the extended MAF file will not all be shown in cBioPortal but only loaded into the database. Maybe assign a superscripted number or symbol behind each column that will be displayed in cBioPortal to make it more clear.

Comment (sandertan) [reply docs/File-Formats.md]: Done. What do you think?

File: docs/File-Formats.md (hunk 1 above)
Comment (sandertan) [reply docs/File-Formats.md]: Done

File: docs/File-Formats.md (hunk 2 above)
Comment (dionnezaal) [reply docs/File-Formats.md]: I think it looks great. Not distracting and clearly gives the extra information.

**Comment by sandertan** [comment]
Thanks. It's squashed and rebased now. @pieterlukasse do you want to take a look at it?

**Comment by sandertan** [comment]
Rebased it again on master, and updated the screenshot for custom driver annotations with screenshot from OncoPrint in v1.11.2.

**Pull request merged**